tests/integration/test_simulation.py

```diff
@@ -1,459 +0,0 @@
-        assert itask.mode_settings.sim_task_fails is True
-
-        # Let task finish.
-        monkeytime(itask.mode_settings.timeout + 1)
-        assert sim_time_check(
-            schd.task_events_mgr, [itask],
-            schd.workflow_db_mgr
-        ) is True
-
-        # The mode_settings object has been cleared:
-        assert itask.mode_settings is None
-        # Change a setting using broadcast:
-        schd.broadcast_mgr.put_broadcast(
-            ['1066'], ['one'], [{
-                'simulation': {'fail cycle points': ''}
-            }])
-        # Submit again - result is different:
-        schd.task_job_mgr._simulation_submit_task_jobs(
-            [itask], schd.workflow)
-        assert itask.mode_settings.sim_task_fails is False
-
-        # Assert Clearing the broadcast works
-        schd.broadcast_mgr.clear_broadcast()
-        schd.task_job_mgr._simulation_submit_task_jobs(
-            [itask], schd.workflow)
-        assert itask.mode_settings.sim_task_fails is True
-
-        # Assert that list of broadcasts doesn't change if we submit
-        # Invalid fail cycle points to broadcast.
-        itask.mode_settings = None
-        schd.broadcast_mgr.put_broadcast(
-            ['1066'], ['one'], [{
-                'simulation': {'fail cycle points': 'higadfuhasgiurguj'}
-            }])
-        schd.task_job_mgr._simulation_submit_task_jobs(
-            [itask], schd.workflow)
-        assert (
-            'Invalid ISO 8601 date representation: higadfuhasgiurguj'
-            in log.messages[-1])
-
-        schd.broadcast_mgr.put_broadcast(
-            ['1066'], ['one'], [{
-                'simulation': {'fail cycle points': '1'}
-            }])
-        schd.task_job_mgr._simulation_submit_task_jobs(
-            [itask], schd.workflow)
-        assert (
-            'Invalid ISO 8601 date representation: 1'
-            in log.messages[-1])
-
-        # Broadcast tasks will reparse correctly:
-        schd.broadcast_mgr.put_broadcast(
-            ['1066'], ['one'], [{
-                'simulation': {'fail cycle points': '1945, 1977, 1066'},
-                'execution retry delays': '3*PT2S'
-            }])
-        schd.task_job_mgr._simulation_submit_task_jobs(
-            [itask], schd.workflow)
-        assert itask.mode_settings.sim_task_fails is True
-        assert itask.try_timers['execution-retry'].delays == [2.0, 2.0, 2.0]
-        # n.b. rtconfig should remain unchanged, lest we cancel broadcasts:
-        assert itask.tdef.rtconfig['execution retry delays'] == [5.0, 5.0]
-
-
-async def test_db_submit_num(
-    flow, one_conf, scheduler, run, complete, db_select
-):
-    """Test simulation mode correctly increments the submit_num in the DB."""
-    schd: Scheduler = scheduler(flow(one_conf), paused_start=False)
-    async with run(schd):
-        await complete(schd, '1/one')
-    assert db_select(schd, False, 'task_states', 'submit_num', 'status') == [
-        (1, 'succeeded'),
-    ]
```

Comment:
This test has not made it over to `tests/integration/run_modes/test_simulation.py`

Reply:
Well spotted!